### ISSUE TITLE
CRAN + JOSS submission

### ISSUE THREAD

**Issue by seabbs** [epiforecasts/RtD3#17]
It would be nice to push this to CRAN and [JOSS](https://joss.theoj.org) if possible as a way of gaining a user base. What do you think @hamishgibbs - interested?

**Comment by hamishgibbs** [comment]
Yes interested. But not familiar with CRAN submission process. I will do some reading and we can discuss Monday?

**Comment by seabbs** [comment]
How do you think we are doing on the CRAN target? The buttons are looking pretty fly 😉

**Comment by hamishgibbs** [comment]
Yes - buttons have remained in place. Let's submit to CRAN.

**Comment by seabbs** [comment]
The only thing to maybe do is #42  but if that is a real pain (it certainly was for the ggplots) maybe leave for later?

**Comment by seabbs** [comment]
If ready to go then to release to cran there is a devtools function called `devtools::release` that takes you through the required checks. 

The first step is to run `devtools::check_rhub` and `devtools::check_win_hub` (not sure second one is quite correct) to check everything is working on their CI. 

As far as I can see everything is looking good.

**Comment by hamishgibbs** [comment]
Sounds good - I found this to walk through the process: https://r-pkgs.org/release.html.

**Comment by seabbs** [comment]
Looks spot on - it should be fairly easy 🤞

**Comment by sbfnk** [comment]
Closing as repository is now archived.
